ores_celery.py

```diff
@@ -22,5 +22,4 @@
 application = score_processor.application
 
 if __name__ == '__main__':
-    logging.getLogger('ores').setLevel(logging.DEBUG)
-    application.worker_main(argv=["celery_worker", "--loglevel=INFO"])
+    application.worker_main(argv=["celery_worker"])
```

Comment:
By setting the log level up in __main__, we can see more debugging when running tests.  Why get rid of that?